packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -88,5 +95,64 @@ test.describe('`Pie Modal is closed`', () => {
             expect(messages).toHaveLength(1);
         });
     });
-});
 
+    test('should return focus to specified element', async ({ mount, page }) => {
```

Comment:
Does ESC need a test too as that should also focus back?

Reply:
Nice spot. This has actually opened a small can of worms which I need to debug 😄